homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -68,13 +68,50 @@ class SwissPublicTransportSensorEntityDescription(SensorEntityDescription):
             key=f"departure{i or ''}",
             translation_key=f"departure{i}",
             device_class=SensorDeviceClass.TIMESTAMP,
+            icon="mdi:bus-clock",
             has_legacy_attributes=i == 0,
             value_fn=lambda data_connection: data_connection["departure"],
             exists_fn=lambda data_connection: data_connection is not None,
             index=i,
         )
         for i in range(SENSOR_CONNECTIONS_COUNT)
     ],
+    SwissPublicTransportSensorEntityDescription(
+        key="duration",
+        translation_key="duration",
```

Comment:
This one can have the translation key omitted as the device class will take care of the icon and name